resources/public/css/compiled/styles.css

```diff
@@ -1 +1 @@
portant}}@media(max-width:767px){.visible-xs-block{display:block !important}}@media(max-width:767px){.visible-xs-inline{display:inline !important}}@media(max-width:767px){.visible-xs-inline-block{display:inline-block !important}}@media(min-width:768px) and (max-width:991px){.visible-sm{display:block !important;display:table !important;display:table-row !important;display:table-cell !important}table.visible-sm tr.visible-sm th.visible-sm td.visible-sm}@media(min-width:768px) and (max-width:991px){.visible-sm-block{display:block !important}}@media(min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline !important}}@media(min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block !important}}@media(min-width:992px) and (max-width:1199px){.visible-md{display:block !important;display:table !important;display:table-row !important;display:table-cell !important}table.visible-md tr.visible-md th.visible-md td.visible-md}@media(min-width:992px) and (max-width:1199px){.visible-md-block{display:block !important}}@media(min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline !important}}@media(min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block !important}}@media(min-width:1200px){.visible-lg{display:block !important;display:table !important;display:table-row !important;display:table-cell !important}table.visible-lg tr.visible-lg th.visible-lg td.visible-lg}@media(min-width:1200px){.visible-lg-inline{display:inline !important}}@media(min-width:1200px){.visible-lg-inline-block{display:inline-block !important}}@media(max-width:767px){.hidden-xs{display:none !important}}@media(min-width:768px) and (max-width:991px){.hidden-sm{display:none !important}}@media(min-width:992px) and (max-width:1199px){.hidden-md{display:none !important}}@media(min-width:1200px){.hidden-lg{display:none !important}}@media print{.visible-print{display:block !important}th.visible-print,td.visible-print{display:table-cell !important}table.visible-print{display:table !important}tr.visible-print{display:table-row !important}}.visible-print-block{display:none !important}@media print{.visible-print-block{display:block !important}}.visible-print-inline{display:none !important}@media print{.visible-print-inline{display:inline !important}}.visible-print-inline-block{display:none !important}@media print{.visible-print-inline-block{display:inline-block !important}}@media print{.hidden-print{display:none !important}}@media(max-width:767px){.user-icon{display:none}.list-character-summary{font-size:18px}.character-summary{flex-wrap:wrap}.app-header{height:auto;background-image:none;background-color:rgba(0,0,0,0.3);min-height:0}.app-header-bar{min-height:50px;backdrop-filter:none;-webkit-backdrop-filter:none}.content{width:100%}.header-button-text,.header-links{display:none}}@media(max-width:1199px){.registration-image{display:none}.registration-content{width:100%;height:100%}.registration-input{width:100%}}
```

Comment:
This file just needs to be removed from source, it is auto built.

Reply:
So should we add it to the gitignore or just delete it?

Reply:
Just remove it from tracked code...